powertools-parameters/src/test/java/software/amazon/lambda/powertools/parameters/DynamoDbProviderTest.java

```diff
@@ -0,0 +1,167 @@
+package software.amazon.lambda.powertools.parameters;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.*;
+import software.amazon.lambda.powertools.parameters.cache.CacheManager;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+public class DynamoDbProviderTest {
+
+    @Mock
+    DynamoDbClient client;
+
+    @Captor
+    ArgumentCaptor<GetItemRequest> getItemValueCaptor;
+
+    @Captor
+    ArgumentCaptor<QueryRequest> queryRequestCaptor;
+
+
+    private DynamoDbProvider provider;
+    private final String tableName = "ddb-test-table";
+
+    @BeforeEach
+    public void init() {
+        openMocks(this);
+        CacheManager cacheManager = new CacheManager();
+        provider = new DynamoDbProvider(cacheManager, client, tableName);
+    }
+
+
+    @Test
+    public void getValue() {
+
+        // Arrange
+        String key = "Key1";
+        String expectedValue = "Value1";
+        HashMap<String, AttributeValue> responseData = new HashMap<String, AttributeValue>();
+        responseData.put("id", AttributeValue.fromS(key));
+        responseData.put("val", AttributeValue.fromS(expectedValue));
```

Comment:
we should use "value" instead, to be compatible with python: https://awslabs.github.io/aws-lambda-powertools-python/2.9.1/utilities/parameters/#dynamodbprovider